packages/model/Cube.ts

```diff
@@ -6,6 +6,10 @@ import { dcterms, schema, xsd } from '@tpluscode/rdf-ns-builders'
 import { cc, cube } from '@cube-creator/core/namespace'
 import { initializer } from './lib/initializer'
 import { NamedNode } from 'rdf-js'
+import { namedNode } from '@rdf-esm/data-model'
+
+export const Draft = namedNode('https://ld.admin.ch/definedTerm/CreativeWorkStatus/Draft')
+export const Published = namedNode('https://ld.admin.ch/definedTerm/CreativeWorkStatus/Published')
```

Comment:
We have a separate point on the roadmap. Might asa well make the change here. @martinmaillard ?

Reply:
I guess it depends how it affects existing data/applications

Reply:
If only we took advantage of :)

```turtle
<https://ld.admin.ch/vocabulary/CreativeWorkStatus/Published>
  owl:sameAs <https://ld.admin.ch/definedTerm/CreativeWorkStatus/Published> .
```

Reply:
But agree in principle. Let's address this separately and look more carefully at the consequences

Reply:
`sameAs` is nice but... we don't use inference for anything as far as I know. So it would require every app that uses any property to also check `sameAs` everywhere. If I'm not mistaken, that wouldn't help much here. Unless there's a magical thing I don't know about. I would be very happy to learn about it ;)